src/js/base/ripe.js

```diff
@@ -1623,13 +1623,17 @@ ripe.Ripe.prototype._guessURL = async function() {
 /**
  * @ignore
  */
-ripe.Ripe.prototype._initBundles = async function(locale = null, defaultLocale = "en_us") {
-    locale = locale || this.locale || defaultLocale;
-    const globalBundleP = this.localeBundleP(locale, "scales");
-    const sizesBundleP = this.localeBundleP(locale, "sizes");
-    const [globalBundle, sizesBundle] = await Promise.all([globalBundleP, sizesBundleP]);
-    this.addBundle(globalBundle, locale);
-    this.addBundle(sizesBundle, locale);
+ripe.Ripe.prototype._initBundles = async function(locale = this.locale, defaultLocale = "en_us") {

```

Comment:
I'm not sure you can access this within the default value initialization... At least in Python it would not work

Reply:
I believe so, tested locally and is a working solution